helm-charts/codegen-openshift/README.md

```diff
@@ -0,0 +1,34 @@
+# CodeGen
+
+Helm chart for deploying CodeGen service on Red Hat OpenShift.
+
+# Prerequisites
+1. Red Hat OpenShift Cluster with dynamic StorageClass to provision PersistentVolumes e.g. OpenShift Data Foundation)
+2. Account on https://huggingface.co/, access to model *ise-uiuc/Magicoder-S-DS-6.7B* (for Xeon) or *meta-llama/CodeLlama-7b-hf* (for Gaudi) and token with Read permissions.
+
```

Comment:
I suggest Image Registry as one of prereqs

Reply:
Added